cob_kinematics/ikfast/plugins.xml

```diff
@@ -0,0 +1,14 @@
+<class_libraries>
+<library path="ros/lib/libikfast_lwa">
+<class name="cob_kinematics/IKFast_lwa" type="lwa::IKFastPlugin" base_class_type="kinematics::KinematicsBase"/>
+</library>
+<library path="ros/lib/libikfast_ur10">
+<class name="cob_kinematics/IKFast_ur10" type="ur10::IKFastPlugin" base_class_type="kinematics::KinematicsBase"/>
+</library>
+<library path="ros/lib/libikfast_lbr">
+<class name="cob_kinematics/IKFast_lbr" type="lbr::IKFastPlugin" base_class_type="kinematics::KinematicsBase"/>
+</library>
+<library path="ros/lib/libikfast_ur5">
+<class name="cob_kinematics/IKFast_ur5" type="ur5::IKFastPlugin" base_class_type="kinematics::KinematicsBase"/>
+</library>
+</class_libraries>
```

Comment:
you need to be cautious when using `git add -a` or `git commit -a`...I do not recommend it....
remove this file...